app/controllers/api/v1/notes_controller.rb

```diff
@@ -0,0 +1,37 @@
+module Api
+  module V1
+    class NotesController < ApplicationController
+      def index
+        render json: notes_paged, status: :ok, each_serializer: IndexNoteSerializer
+      end
+
+      def show
+        render json: show_note, status: :ok, serializer: NoteSerializer
```

Comment:
```suggestion
        render json: show_note, status: :ok
```
Si no me equivoco, Rails toma por default el Serializer que se llame igual al modelo. Podemos probarlo?